pkg/terraform/stages/platform/stages.go

```diff
@@ -7,10 +7,9 @@ import (
 	"github.com/openshift/installer/pkg/terraform/stages/compat"
 	"github.com/openshift/installer/pkg/terraform/stages/gcp"
 	"github.com/openshift/installer/pkg/terraform/stages/ibmcloud"
-	awstypes "github.com/openshift/installer/pkg/types/aws"
```

Comment:
I assume this is a wrong conflict resolution after the rebase.
Please add these imports back, it causes the compilation to fail:
```
pkg/terraform/stages/platform/stages.go:18:7: undefined: awstypes
pkg/terraform/stages/platform/stages.go:20:7: undefined: azuretypes
pkg/terraform/stages/platform/stages.go:22:7: undefined: azuretypes
pkg/terraform/stages/platform/stages.go:24:7: undefined: gcptypes 
```